codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker.java

```diff
@@ -57,6 +60,10 @@
 
     private volatile boolean handshakeComplete;
 
+    private volatile long forceCloseTimeoutMillis = -1;
```

Comment:
why is this volatile ?

Reply:
Previously, I had an API to change this value after a handshaker is already created. As far as I removed that, I can remove `volatile` either.